src/ape/managers/chain.py

```diff
@@ -403,6 +410,11 @@ def _contract_types_cache(self) -> Path:
         network_name = self._network.name.replace("-fork", "")
         return self._network.ecosystem.data_folder / network_name / "contract_types"
 
+    # I don't love this. I think I'd rather it be at the config level.
```

Comment:
will others part of the code need it? If no, we can leave it for now.
keeping it underscored will allow us to change it later if needbe